core/consensus/babe/impl/babe_impl.cpp

```diff
@@ -235,8 +235,8 @@ namespace kagome::consensus::babe {
     synchronizer_->syncByBlockHeader(
         announce.header,
         peer_id,
-        [wp = weak_from_this()](
-            outcome::result<primitives::BlockInfo> block_res) {
+        [wp = weak_from_this(), announce = announce](
```

Comment:
```suggestion
        [wp = weak_from_this(), announce](
```

Reply:
ono i tak skopiruetsya

Reply:
No, it must to capture by value for being moved at call, but it is reference in external scope.